commands/plugin_get.go

```diff
@@ -31,9 +33,11 @@ func init() {
 		RunE:    runPluginGetCmd,
 	}
 
+	clientArch, operatingSystem := env.GetClientArch()
```

Comment:
I mentioned this won't work with Windows, because uname isn't supported.

We need to check if GOOS is windows (whatever the literal value is) before running this.

Reply:
It checks if GOOS is windows now first.